android/src/app/organicmaps/widget/placepage/PlacePageWikipediaFragment.java

```diff
@@ -49,6 +52,7 @@ public void onViewCreated(@NonNull View view, @Nullable Bundle savedInstanceStat
 
     mPlaceDescriptionView = view.findViewById(R.id.poi_description);
     View placeDescriptionMoreBtn = view.findViewById(R.id.more_btn);
+    mPlaceDescriptionDescriptionViewContainer = view.findViewById(R.id.poi_description_container);
```

Comment:
Are mPlaceDescriptionView and mPlaceDescriptionDescriptionViewContainer connected?
I suspect that we can keep container only, no?

Reply:
mPlaceDescriptionView is inside mPlaceDescriptionViewContainer (among with a button, separator and others).  It's enough just to hide the container.